notebooks/08-Packages-Modules.ipynb

```diff
@@ -0,0 +1,842 @@
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "### Python and Jupyter Overview"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "- We're working with Python through Jupyter, the most common IDE for data science."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "### Fundamentals"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "- Python's common *atomic*, or basic, data types are:\n",
+    "    - Integers\n",
+    "    - Floats (decimals)\n",
+    "    - Strings\n",
+    "    - Booleans"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "- These simple types can be combined to form more complex types, including:\n",
+    "    - Lists: Ordered collections\n",
+    "    - Dictionaries: Key-value pairs\n",
+    "    - DataFrames: Tabular datasets"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Packages (aka *Modules*)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "So far we've seen several data types that Python offers out-of-the-box.\n",
+    "However, to keep things organized, some Python functionality is stored in standalone *packages*, or libraries of code.\n",
+    "The word \"module\" is generally synonymous with \"package,\" and \"library\"; you will hear all three in discussions of Python."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "If you want more clear definitions, the three can thought of this way:\n",
+    "\n",
+    "- A **package** refers to source code that is bundled in a way that can be distributed to users\n",
+    "  - You can install packages using `pip install package_name`\n",
+    "- A **library** refers to a package that has been installed to a centralized location on an operating system\n",
+    "  - You can view your libraries using `pip list` or `conda list`\n",
+    "- A **module** refers to code that you can import from outside your current script or notebook\n",
+    "  - To use code from a module, you'll need to `import module_name`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "For example, functionality related to the operating system -- such as creating files and folders -- is stored in a package called `os`.\n",
```

Comment:
This comment actually contradicts the preceding slide, since `os` actually isn't something you can download on PyPI. It's pedantic, but given that we've just given the definitions, this could raise (reasonable) questions. I think we should probably call it a "library" or "built-in library".